### ISSUE TITLE
Storage costs

### ISSUE THREAD

**Issue by chartgerink** [ubvu/open-handbook#97]

**Comment by chartgerink** [comment]
Does that mean this topic is no longer necessary/desired? 😊 Happy to close this upon confirming that!

**Comment by peer35** [comment]
This is the page: https://vu.nl/en/employee/research-data-support/research-archiving-storage-cost-model

**Comment by peer35** [comment]
Yup, all 3 topic pages provide info under "Are there costs involved?". I am going to close this one.
